28-04/task_2/task_2.rb

```diff
@@ -0,0 +1,8 @@
+File.open("palindrom.txt").each do |line|
```

Comment:
These two lines of code are the same as in next file. Let's try to create a new class for them called `CleanFile`.
Inside `initialize` method you can pass `file_name`, by default it can be `"palindrom.txt"`. Then create other method called `open` inside this class, and put these two lines of code. It will open the file and clear the content. The last part is to add `yield(str_punct_marks_off_downcase)` on the next line.

After that you can `require` this class and call inside block `{ |str| palindrome?(str) }`